packages/config/src/handlers/informantSMSNotifications/createInformantSMSNotification/handler.ts

```diff
@@ -22,15 +21,10 @@ export default async function createInformantSMSNotification(
 ) {
   const informantSMSNotificationPayload =
     request.payload as IInformantSMSNotificationsModel[]
-  try {
-    await InformantSMSNotification.insertMany(informantSMSNotificationPayload)
-  } catch (e) {
-    throw internal(e.message)
-  }
-  const informantSMSNotifications = await getInformantSMSNotificationsHandler(
-    request,
-    h
-  )
+
+  await InformantSMSNotification.insertMany(informantSMSNotificationPayload)
+
+  const informantSMSNotifications = await getInformantSMSNotificationsHandler()
```

Comment:
I added `strict: true` to tsconfig, and the "e" was any, so the catch phrase was throwing a TypeScript error. Please check these changes where I remove some try-catches. Am I correct in understanding that rethrowing an internal server error in these cases is redundant?

Reply:
Should be equivalent, yes 👍